src/components/visualizations/chart/highcharts/plugins/test/getTickPositions.spec.ts

```diff
@@ -555,8 +589,8 @@ describe('get tick positions for dual axes chart', () => {
                 });
             });
 
-            it('left tick position is empty with out-of-range min/max,' +
-                'right tick position is empty with min > max,', () => {
+            // tslint:disable-next-line:max-line-length
+            it('should left tick position be empty with out-of-range min/max, right tick position be empty with min > max,', () => {
```

Comment:
This sentence doesn't make sense in english :(
What about something like: `it('should return empty left tick position if min/max is out of range; and empty right tick position if min > max')`?
Please review recommended sentence above if it makes sense in this context.